index.html

```diff
@@ -167,6 +167,33 @@ <h2>About<br> Myself</h2>
             </li>
         </ul>
     </section>
+
+    <section class="contact-page">
+        <h2 class="form-heading">Contact me</h2>
+        <p class="form-intro">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+     <form class="contact-form" action="https://formspree.io/f/mayvlell" method="post">
+      <ul>
+        <li>
+            <label for="name"></label>
+            <input class="contact-input" type="text" id="name" name="user-name" required maxlength="30" placeholder="River">
+        </li>
+
+        <li>
+            <label for="email"></label>
+            <input class="contact-input" type="email" id="email" name="user-email" required placeholder="melidypond@mail.com">
+        </li>
+
```

Comment:
- [ ] **[OPTIONAL]**
- The HTML label `for` attribute specifies which form element a `label` is bound to so that when a user enters any input in your form, it knows and differentiates the inputs values when submitted to the server. 
Kindly correct it to have the same value as the input name attribute value.